install.sh

```diff
@@ -14,9 +14,11 @@ cat 90forceyes >> /etc/yum.conf
 touch "$HOME/.ssh/known_hosts"
 mkdir -p /etc/drydock
 
-echo "================= Installing basic packages ==================="
+echo "================= Installing basic packages ===================="
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 
```

Comment:
which key is this? can you add a comment here so that we know which key is being imported